packages/device/src/classes/device.ts

```diff
@@ -95,12 +103,9 @@ export class Device extends EventEmitter {
       'castmill-device',
       opts?.cache?.maxItems || 1000
     );
-    this.resourceManager = new ResourceManager(this.cache);
-
-    this.contentQueue = new Playlist('content-queue', this.resourceManager);
 
-    const intro = getCastmillIntro(this.resourceManager);
-    this.contentQueue.add(intro);
+    //const intro = getCastmillIntro(this.resourceManager);
```

Comment:
Can this be removed or should we keep it for later?

Reply:
For later. I discovered we did have a default intro content that was static in the player, this may be useful in the future I think, not sure. The interesting thing is that it was not being played, something must have broken along the updates.